examples/AlwaysSucceeds.purs

```diff
@@ -25,37 +27,107 @@ import Contract.Transaction
   , submit
   )
 import Contract.TxConstraints as Constraints
+import Contract.Utxos (utxosAt)
 import Contract.Value as Value
 import Contract.Wallet (mkNamiWalletAff)
+import Data.Array (head)
 import Data.BigInt as BigInt
+import Data.Map as Map
+import Data.Time.Duration (Milliseconds(Milliseconds))
+import Effect.Aff (delay)
+import Plutus.Types.Transaction (UtxoM(UtxoM))
+import Types.Scripts (ValidatorHash)
+import Types.Transaction (TransactionInput(..), TransactionHash)
+import Types.TxConstraints (TxConstraints)
 
 main :: Effect Unit
 main = launchAff_ $ do
   wallet <- Just <$> mkNamiWalletAff
   cfg <- over ContractConfig _ { wallet = wallet } <$> traceContractConfig
   runContract_ cfg $ do
     logInfo' "Running Examples.AlwaysSucceeds"
-    validator <- liftContractM "Invalid script JSON" $ alwaysSucceedsScript
+    validator <- liftContractM "Invalid script JSON" alwaysSucceedsScript
     vhash <- liftContractM "Couldn't hash validator" $ validatorHash validator
-    let
-      -- Note that CTL does not have explicit equivalents of Plutus'
-      -- `mustPayToTheScript` or `mustPayToOtherScript`, as we have no notion
-      -- of a "current" script. Thus, we have the single constraint
-      -- `mustPayToScript`, and all scripts must be explicitly provided to build
-      -- the transaction (see the value for `lookups` below as well)
-      constraints :: Constraints.TxConstraints Unit Unit
-      constraints = Constraints.mustPayToScript vhash unitDatum
-        $ Value.lovelaceValueOf
-        $ BigInt.fromInt 2_000_000
-
-      lookups :: Lookups.ScriptLookups PlutusData
-      lookups = Lookups.validator validator
-
-    ubTx <- liftedE $ Lookups.mkUnbalancedTx lookups constraints
-    BalancedSignedTransaction bsTx <-
-      liftedM "Failed to balance/sign tx" $ balanceAndSignTx ubTx
-    txId <- submit bsTx.signedTxCbor
-    logInfo' $ "Tx ID: " <> show txId
+    logInfo' "Attempt to lock value "
+    txId <- payToAlwaysSucceeds vhash validator
+    countToZero 20
+    logInfo' "Try to spend locked values"
+    spendFromAlwaysSucceeds vhash validator txId
+
+countToZero :: Int -> Contract () Unit
+countToZero n =
+  if n <= 0 then
+    pure unit
+  else do
+    logInfo' $ "Waiting before we try to unlock : " <> show n
+    liftAff <<< delay $ Milliseconds 1000.0
```

Comment:
`Milliseconds` has a `Newtype` instance, so you can use `wrap` instead of the constructor. Also saves you an import

Reply:
I keep forgetting we have the `Newtype` class in PS, is just nice to have it!